.eslintrc

```diff
@@ -14,3 +14,4 @@ rules:
   no-use-before-define: [warn, { functions: false }]
   react/no-multi-comp: [off]
   no-throw-literal: [off]
+  import/no-unresolved: [warn]
```

Comment:
I can reproduce the problem here. Maybe try removing `node_modules`, and reinstalling based on #15?